service/account.go

```diff
@@ -0,0 +1,115 @@
+package service
+
+import (
+	"ApniUniversity/models"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func (s *Service) AddAccount(account *models.Account) (int, error) {
+	accounts, err := s.db.GetAccounts()
+	if err != nil {
+
+		return 0, err
+	}
+
+	account.ID = accounts[len(accounts)-1].ID + 1
+
+	return s.db.AddOrUpdateAccount(account)
+}
+
+func (s *Service) GetAccounts() ([]*models.Account, error) {
+
+	return s.db.GetAccounts()
+}
+
+func (s *Service) UpdateAccount(account *models.Account) (int, error) {
+	_, err := s.db.GetAccount(account.ID)
+	if err != nil {
+
+		return 0, err
+	}
+
+	if account.AccountType == "teacher" {
```

Comment:
create constants for teacher,student etc since they wil be used throughtout the program it is better to define them once and use again and again

Reply:
should i declare them in the respective micro-service because these are only used in accounts service?